README.md

```diff
@@ -1,37 +1,47 @@
-# TASK2
+# TASK3
 
-#### Build image 
+## Use maven wrapper
+
+Open your project package in cmd
 
 ```
-docker build -t webserviceimage .
+./mvnw.cmd clean package
 ```
 
-#### Run docker compose
+## You have two options to launch application
+
+### First option
+
+##### Run docker compose
 
 ```
 docker-compose run --service-ports -e POSTGRES_PASSWORD=[POSTGRES_PASSWORD] -e POSTGRES_USER=[POSTGRES_USER] -e POSTGRES_URL=jdbc:postgresql://db:5432/postgres web
 ```
-There is default name of your database :"postgres"
-#### Request
+##### Request
 http://localhost:8080/actuator/health
-#### You must see
+###### You must see
 ```
 {"status":"UP"}
 ```
-#### To add user send request to 
-http://localhost:8080/users
-####
- ```
- {
-     "firstName": "[userFirstName]",
-     "secondName": "[userSecondName]"
- }
- ```
-#### To get user by id send request to 
-http://localhost:8080/users/{id}
-### To get all users send request to
-http://localhost:8080/users
+#### Now you can check all possible requests with swagger
 
+##### http://localhost:8080/swagger-ui.html
 
+There is default name of your database :"postgres"
+### Second option
+#### Kubernetes config
+##### Before starting deployments you need to set database configuration
 
+##### Set your database user name and password
+```
+kubectl create secret generic user --from-literal=POSTGRES_USER=[YOUR POSTGRES USER] --from-literal=POSTGRES_PASSWORD=[YOUR POSTGRES PASSWORD]
```

Comment:
README file not updated. 
Also, use same convention for configmaps also. postgres-conf =\